frontend/src/app/images/images.component.ts

```diff
@@ -1,31 +1,38 @@
-import { Component, OnInit } from '@angular/core';
+import { Component } from '@angular/core';
 
 @Component({
   selector: 'app-images',
   templateUrl: './images.component.html',
   styleUrls: ['./images.component.css']
 })
-export class ImagesComponent implements OnInit {
-  clickMessage = null;
-  clickMessageTimeout = null;
-
-  constructor() { }
-
-  ngOnInit() {
-  }
+export class ImagesComponent {
+  clickMessage = undefined;
+  clickMessageTimeout = undefined;
 
   onClickBoy() {
+    // TODO: Implement logic.
     clearTimeout(this.clickMessageTimeout);
-    this.clickMessage = "You clicked Boy.";
+    this.clickMessage = 'You clicked Boy.';
 
-    this.clickMessageTimeout = setTimeout(() => { this.clickMessage = null }, 2000);
+    this.resetClickMessageTimeout();
   }
 
   onClickGirl() {
+    // TODO: Implement logic.
     clearTimeout(this.clickMessageTimeout);
-    this.clickMessage = "You clicked Girl.";
+    this.clickMessage = 'You clicked Girl.';
```

Comment:
Noice, I might need to start adding linter rules in this case so we don't accidentally mix single and double quotes across the project.